src/main/java/examples/NetExamples.java

```diff
@@ -598,6 +598,11 @@ public void example50(Vertx vertx) throws CertificateException {
       .listen(8080);
   }
 
+  public void example51(Vertx vertx) {
+    NetServerOptions options = new NetServerOptions().setUseProxyProtocol(true);
+    NetServer server = vertx.createNetServer(options);
```

Comment:
I think we should add:

```java
server.connectHandler(so -> {
  // Print the actual client address provided by the HA proxy protocol instead of the proxy address
  System.out.println(so.remoteAddress());
});
```